src/test/java/org/elasticsearch/action/count/CountRequestBuilderTests.java

```diff
@@ -78,7 +78,7 @@ public void testStringQueryToString() {
         CountRequestBuilder countRequestBuilder = client.prepareCount();
         String query = "{ \"match_all\" : {} }";
         countRequestBuilder.setQuery(new BytesArray(query));
-        assertThat(countRequestBuilder.toString(), containsString("\"query\":{ \"match_all\" : {} }"));
+        assertThat(countRequestBuilder.toString(), containsString("\"query\" : {\n    \"match_all\" : { }\n  }"));
```

Comment:
why does it need to have line breaks now compared to before? Wouldn't this test fail on windows?